### PR TITLE
chore(deps): update dependencies

### PR DESCRIPTION
**Pull request by anehx**
Depends on #13 

Closes #8 
Closes #10 
Closes #15

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
   },
   "devDependencies": {
     "@semantic-release/changelog": "^3.0.2",
-    "husky": "1.3.1"
+    "husky": "^2.3.0"
```
Comment (open-dynaMIX) [inline package.json]: Renovate will pin this again ([config](https://github.com/projectcaluma/renovate-config/blob/master/renovate.json#L36)).

[This just happened with the line above](https://github.com/adfinis-sygroup/semantic-release-config/pull/15/files).

Comment (anehx) [reply package.json]: In that case we should maybe pin all of the dependencies... This was the only pinned dependency. Right?

Comment (open-dynaMIX) [reply package.json]: If you really want to use a semver range, you need to set this up in the renovate config.

Right now for dev deps it's:

```
      {
        "depTypeList": ["devDependencies"],
        "rangeStrategy": "pin"
      },
```

Comment (anehx) [reply package.json]: I think it's best if we use pinned deps. Also I moved the changelog plugin to normal dependencies

Comment (open-dynaMIX) [reply package.json]: :+1: Did you forget to push?

Comment (anehx) [reply package.json]: I amended the commit :wink:

Comment (open-dynaMIX) [reply package.json]: I see the force push, but it seems to have no changes....

Comment (anehx) [reply package.json]: Turns out amending does not help if you forget to add the changes :joy: Should be pushed now

Comment (open-dynaMIX) [reply package.json]: :wink:

But it's still `"husky": "^2.3.0"`.

Comment (open-dynaMIX) [reply package.json]: We had a race condition. All good :smile:

**Pull request merged**